src/subject_name/dns_name.rs

```diff
@@ -55,9 +55,12 @@ impl AsRef<str> for DnsName {
 }
 
 /// Requires the `alloc` feature.
-// Deprecated
+/// Deprecated.
 #[cfg(feature = "alloc")]
 impl From<DnsNameRef<'_>> for DnsName {
+    // TODO(XXX): Remove this trait impl in the next release. We can't mark it as
```

Comment:
Did you try to deprecate the method or the trait impl (or both)? I thought impl deprecations were a thing.

Reply:
I tried both and the warning persisted.